src/compass/utils/validate_product.py

```diff
@@ -10,7 +10,7 @@
 from osgeo import gdal
```

Comment:
```suggestion
from osgeo import gdal

from compass.utils.h5_helpers import DATA_PATH
```
Use common path from h5_helpers